scipy/stats/tests/test_distributions.py

```diff
@@ -3694,6 +3694,40 @@ def test_skewcauchy_R(self):
         assert_allclose(stats.skewcauchy.ppf(cdf, a), x)
 
 
+class TestJFSkewT:
+    def test_stats(self):
+        actual = stats.jf_skew_t(8, 4).stats('mvsk')
+        expected = [
+            1.4247801402401363,
+            1.684287266263012,
+            0.7983625393780869,
+            2.3345439210504404,
+        ]
+        assert_allclose(actual, expected)
+
+    def test_cdf_to_pdf(self):
+        q = [0.0, 0.1, 0.25, 0.75, 0.90, 1.0]
+        x = stats.jf_skew_t(8, 4).ppf(q)
+        q_out = stats.jf_skew_t(8, 4).cdf(x)
+        assert_allclose(q, q_out)
```

Comment:
```suggestion
```

These are already covered in the generic tests: https://github.com/scipy/scipy/blob/main/scipy/stats/tests/test_continuous_basic.py#L580

Reply:
Thanks, I'll remove. I got the idea from the test here: https://github.com/scipy/scipy/blob/b5e02b70c9b49d8b6283542be8786f0cdd734f2a/scipy/stats/tests/test_distributions.py#L6858-L6862 

Perhaps there was a specific reason they wanted to test more quantiles? I don't think it's important for this distribution though, so I'll remove.